api/NotificationEvent.json

```diff
@@ -0,0 +1,213 @@
+{
+  "api": {
+    "NotificationEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/NotificationEvent",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": [
+            {
+              "version_added": "42"
+            },
+            {
+              "notes": "Secure context required (HTTPS)",
```

Comment:
Please add a sub feature for this instead of a note
```
 "secure_context_required": {
        "description": "Secure context required (HTTPS)",
```